src/amos/dgamln.jl

```diff
@@ -0,0 +1,238 @@
+const _DGAMLN_GLN = Array{Float64}(100)
+const _DGAMLN_CF = Array{Float64}(22)
+function DGAMLN(Z::Float64,IERR::Int32)
+    __DGAMLN__::Float64 = zero(Float64)
+    const CF = _DGAMLN_CF
+    CON::Float64 = zero(Float64)
+    FLN::Float64 = zero(Float64)
+    FZ::Float64 = zero(Float64)
+    const GLN = _DGAMLN_GLN
+    I::Int32 = zero(Int32)
+    I1M::Int32 = zero(Int32)
+    K::Int32 = zero(Int32)
+    MZ::Int32 = zero(Int32)
+    NZ::Int32 = zero(Int32)
+    RLN::Float64 = zero(Float64)
+    S::Float64 = zero(Float64)
+    T1::Float64 = zero(Float64)
+    TLG::Float64 = zero(Float64)
+    TRM::Float64 = zero(Float64)
+    TST::Float64 = zero(Float64)
+    WDTOL::Float64 = zero(Float64)
+    ZDMY::Float64 = zero(Float64)
+    ZINC::Float64 = zero(Float64)
+    ZM::Float64 = zero(Float64)
+    ZMIN::Float64 = zero(Float64)
+    ZP::Float64 = zero(Float64)
+    ZSQ::Float64 = zero(Float64)
+    begin 
+        GLN[Int32(1)] = 0.0
```

Comment:
These should be moved outside the function.

Reply:
Seems like this function is not used by anything.